generators/client/templates/common/package.json

```
@@ -1,7 +1,10 @@
 {
   "devDependencies": {
     "cypress": "6.6.0",
-    "puppeteer": "8.0.0"
+    "puppeteer": "8.0.0",
+    "babel-loader": "8.2.2",
+    "babel-plugin-istanbul": "6.0.0",
+    "@cypress/code-coverage": "3.9.2"
```

```suggestion
    "@cypress/code-coverage": "3.9.2",
    "babel-loader": "8.2.2",
    "babel-plugin-istanbul": "6.0.0",
    "cypress": "6.6.0",
    "nyc": "15.1.0",
    "puppeteer": "8.0.0"
```
Reorder and add nyc to print the report.